analytics_data_api/v0/views/learners.py

```diff
@@ -168,7 +181,7 @@ def _validate_query_params(self):
 
     def get_queryset(self):
         """
-        Fetches the user list from elasticsearch.  Note that an
+        Fetches the user list from elasticsearch. Note that an
```

Comment:
take it up with DB. He added some mojo that automatically strips out extra spaces and line feeds too improve the quality of the doc files. Sublime must apply it to any file that you edit, though, not just rst's... in short, I didn't take it out on purpose, and I will have to find --and remember to use-- a different editor if you want to keep them in.